mobile/src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java

```diff
@@ -397,6 +400,8 @@ private void tearDownState(Bundle state) {
     public void onDestroyView() {
         super.onDestroyView();
 
+        SharedVariables.isAlertsFragmentAvailable = false;
```

Comment:
@garg-anuj yes but since it's checking against a flag in the same class and since when we switch to Fav it will still try to display the triggers it gets in Alerts view unless we set the flag to false (not set it true at Fav)